tsconfig.json

```diff
@@ -0,0 +1,68 @@
+{
+    "compilerOptions": {
+        /* Basic Options */
+        // "incremental": true,                   /* Enable incremental compilation */
```

Comment:
Can you look for description of this property and maybe we should enable it - because incremental compilation is a great thing to have as far as I know.

Reply:
I am just not sure how it work in TS

Reply:
> Can you look for description of this property and maybe we should enable it - because incremental compilation is a great thing to have as far as I know.

which one? "incremental"?

Reply:
TypeScript 3.4 introduces a new flag called --incremental which tells TypeScript to save information about the project graph from the last compilation. The next time TypeScript is invoked with --incremental, it will use that information to detect the least costly way to type-check and emit changes to your project.